analyses/cell-type-neuroblastoma-04/run-analysis.sh

```diff
@@ -15,6 +18,9 @@ set -euo pipefail
 module_dir=$(dirname "${BASH_SOURCE[0]}")
 cd ${module_dir}
 
+# Define the nbatlas_version
```

Comment:
I think this is confusing me. Don't we want to generate models for both versions and then you are just specifying subset for running through CI? Why don't we just use the `testing` flag to specify only getting the subset version.